cmake/CompilerFlags.cmake

```diff
@@ -163,19 +153,26 @@ else()
     add_warning_flag("-Wmissing-prototypes")
     add_warning_flag("-Wmissing-variable-declarations")
 
-    # Disable some strong warning with clang that are OK here
+    # Disable some strong warning that are OK here
     add_warning_flag("-Wno-unknown-pragmas")
+    add_warning_flag("-Wno-overlength-strings")
```

Comment:
I'm not familiar with this warning. I guess this is a strong warning here?

Reply:
No, this is a warning about strings longer than 4096 characters, which is the minimum a C99 conforming implementation must support. It comes from `-Wpedantic`.